src/test/java/wooteco/subway/path/PathTest.java

```diff
@@ -0,0 +1,59 @@
+package wooteco.subway.path;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import wooteco.subway.line.domain.Line;
+import wooteco.subway.path.domain.Path;
+import wooteco.subway.station.domain.Station;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PathTest {
+    Station 강남역 = new Station(1L, "강남역");
+    Station 양재역 = new Station(2L, "양재역");;
+    Station 교대역 = new Station(3L, "교대역");;
+    Station 남부터미널역 = new Station(4L, "남부터미널역");;
+
+    Line 이호선 = new Line("이호선", "green");
+    Line 삼호선 = new Line("삼호선", "brown");
+    Line 신분당선 = new Line("신분당선", "pink");
+
+    Path path;
```

Comment:
```suggestion
    private final Station 강남역 = new Station(1L, "강남역");
    private final Station 양재역 = new Station(2L, "양재역");
    private final Station 교대역 = new Station(3L, "교대역");
    private final Station 남부터미널역 = new Station(4L, "남부터미널역");

    private final Line 이호선 = new Line("이호선", "green");
    private final Line 삼호선 = new Line("삼호선", "brown");
    private final Line 신분당선 = new Line("신분당선", "pink");

    private Path path;
```

테스트 코드도 프로덕션 코드와 마찬가지로 지속적으로 관리해야 할 대상입니다. 🙂